docs/guests.md

```diff
@@ -241,4 +250,4 @@ If you are using Xen Orchestra, you can switch the "Windows Update tools" advanc
 ... So the "Windows Update tools" option is not a complete solution if you need the guest metrics from the management agent. However it may be a convenient way to get future driver updates if you wish so.
 
 #### Switching from XCP-ng tools to Citrix tools
-If your VM already has XCP-ng tools and you wish to switch to Citrix tools, then you need to do the same kind of clean-up as described higher in this document for the opposite situation.
\ No newline at end of file
+If your VM already has XCP-ng tools and you wish to switch to Citrix tools, then you need to do the same kind of clean-up as described higher in this document for the opposite situation.
```

Comment:
I don't know where this diff comes from. I did not touch that in the online editor.